data-prepper-plugins/kafka-plugins/src/test/resources/sample-pipelines-sink-ssl.yaml

```diff
@@ -0,0 +1,40 @@
+log-pipeline :
+  source :
+    #    sqs:
+    #      acknowledgments: true
+    #      queue_urls:
+    #        - https://sqs.ap-south-1.amazonaws.com/045129910014/data-prepper-testing
+    #      aws:
+    #        sts_region: "ap-south-1"
+    #        sts_role_arn: "arn:aws:iam::XXXXXXX:role/dataprepper"
+    #      polling_frequency: PT0M
+    #      batch_size: 10
+    #      number_of_threads: 1
+    random:
+  sink :
+      kafka-sink:
+        bootstrap_servers:
+          - "pkc-l7pr2.ap-south-1.aws.confluent.cloud:9092"
+        thread_wait_time: 1000
+        dlq:
+          s3:
+            bucket: "mydlqtestbucket"
+            key_path_prefix: "dlq-files/"
+            sts_role_arn: "arn:aws:iam::xxxxx:role/dataprepper"
+            region: "ap-south-1"
+
+        serde_format: plaintext
+        topics:
```

Comment:
Whats the use case of having more than 1 topics in sink? are we going to write same message to multiple topics?

Reply:
yes, its as per the HLD.